pkg/component/component.go

```diff
@@ -880,17 +878,18 @@ func GetComponentDesc(client *occlient.Client, componentName string, application
 		}
 	}
 
-	componentDesc = Description{
-		ComponentName:      componentName,
-		ComponentImageType: componentImageType,
-		Path:               path,
-		Env:                filteredEnv,
-		Storage:            appStore,
-		URLs:               urlList,
-		LinkedComponents:   linkedComponents,
-		LinkedServices:     linkedServices,
-	}
-	return componentDesc, nil
+	currCompo, _ := GetCurrent(applicationName, projectName)
+
+	component = getMachineReadableFormat(componentName, componentType)
+	component.Spec.Source = path
+	component.Spec.URL = urls
+	component.Spec.Storage = storage
```

Comment:
Shouldn't these also be set inside `getMachineReadableFormat` or some other method?

Reply:
so then method will need to provide too much inputs and codeclimate will cry :(

Reply:
Cool